www/docs/configuration.md

```diff
@@ -53,8 +53,7 @@ priority: extra
 
 # Maintainer.
 # Defaults to empty on rpm and apk
-# Leaving this field empty on 'deb' packages is deprecated, and will be removed
-# in a future release
```

Comment:
This is just a drive-by fix to make it more in line with the deprecation warning.